smartsim/_core/_install/buildenv.py

```diff
@@ -199,10 +201,15 @@ def __init__(self, vers):
             self.version = vers
 
     def __getattr__(self, name):
-        return self.defaults[self.version][name]
+        try:
+            return self.defaults[self.version][name]
+        except KeyError:
+            raise AttributeError(
+                f"'{type(self).__name__}' object has no attribute '{name}'"
```

Comment:
If I'm trying to install and I get this error message, am I going to have any idea what's going on? I think we need to give the user something to go on here -- even if it's just "contact the dev team"

Reply:
Good idea!! Ideally this should never bubble up to a user's view, but on the off chance that it does we should probably give them some direction on where to report it. I added links to out GH issues page and to our community page in our docs